### DISCUSSION TITLE
PFC changes to test & support x64 deployment.

### DISCUSSION THREAD

**Discussion by amykonio** [OpenSourcePFCLibraries/2019#34] [Ideas]
Hi.
I would like to see pfc's to be updated so they can be used in 64bit deployment.
I know there are a lot of things to check.
I wonder if we could make a list of what should be ckecked.
I know that we have to review all windows api's and also all variables using - storing handle values.
But is anything else that we should do?

I was thinking if we can make a special release during this year (something like a beta) with support for x64 deployment...

Andreas.

**Comment by brucearmstrong2** [comment]
Those changes have already been done.

**Reply by amykonio** [reply]
Thank you Bruce. I wasn't aware...
